vividus-plugin-accessibility/src/main/java/org/vividus/converter/ParametersToAxeCheckOptionsConverter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.converter;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.Validate;
+import org.jbehave.core.steps.Parameters;
+import org.vividus.accessibility.model.axe.AxeCheckOptions;
+import org.vividus.accessibility.model.axe.AxeOptions;
+import org.vividus.accessibility.model.axe.EnabledProperty;
+import org.vividus.ui.action.ISearchActions;
+
+public class ParametersToAxeCheckOptionsConverter extends AbstractAccessibilityCheckOptionsConverter<AxeCheckOptions>
+{
+    public ParametersToAxeCheckOptionsConverter(ISearchActions searchActions)
+    {
+        super(searchActions);
+    }
+
+    @Override
+    public AxeCheckOptions convertValue(Parameters row, Type type)
+    {
+        String standard = getStandard(row, String.class);
+        List<String> violationsToCheck = getViolationsToCheck(row);
+
+        Validate.isTrue(standard != null || violationsToCheck != null,
+                "Either 'standard' or 'violationsToCheck' must be set");
+
+        Validate.isTrue(standard == null || violationsToCheck == null,
+                "Either 'standard' or 'violationsToCheck' is allowed");
```

Comment:
```suggestion
        Validate.isTrue(standard == null ^ violationsToCheck == null,
                "Either 'standard' or 'violationsToCheck' must be set");
```

Reply:
its shorter but in this case we trade off error message, not sure if its worth it

Reply:
You really don't care about specific messages, you just need to say to user pick either one

Reply:
on standup It was agreed to keep it as is :)